xml_inscriptions/transcribed/MA.Camb.priv.L.Tmp97.6.1.xml

```diff
@@ -119,6 +118,7 @@ changes: correct handling of provenance information
                         <ptr type="rest-of-bibl" target=""/>
                         <biblScope unit="page">00</biblScope>
                         <!-- Hammond, Mason. "An Unpublished Latin Funerary Inscription of Persons Connected with Maecenas." Harvard Studies in Classical Philology vol. 84 (1980): 263-277.https://www.jstor.org/stable/311052
```

Comment:
I'm assuming these are the entries we discussed at the meeting today? If so, let's just make sure to remember to go back and take care of them once we've updated everything.